### ISSUE TITLE
Create re-usable slow-motion library 

### ISSUE THREAD

**Issue by Granjow** [slowmoVideo/slowmoVideo#173]
In order to use slow-motion also in other projects, extract a library which could be used by video editors like [Kdenlive](http://kdenlive.org) for slow-motion effects.

This first needs some coordination with interested projects, for example [mlt](https://github.com/mltframework/mlt), which would use the library.

**Comment by ulatekh** [comment]
There are existing plugins like [frei0r](https://en.wikipedia.org/wiki/Frei0r) that work with kdenlive. Perhaps that'll show how to make slowmoVideo a plugin?

**Comment by Granjow** [comment]
Yes, I have contributed to frei0r as well. The thing is that usually plugins work like this: You give it a frame and it returns a modified frame. Slow motion with curves is quite different as it decides on its own which frame it needs.

Natron looks interesting! Will have a look.

**Comment by AndreaMonzini** [comment]
Hello,
i mentioned slowmoVideo for the Blender new strip retiming, it would be awesome if we could use slowmoVideo in Blender VSE.

https://devtalk.blender.org/t/vse-strip-retiming-feedback/32581/11

For a possible Blender VSE integration the library should be compatible with multi-platform OS.

Thank you
